### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by jdpedrie**
Some proofreading and editing for clarity. I also regenerated the documentation.

One of the larger changes was renaming NEWRULES.md to CONTRIBUTING.md. The lion's share of content in NEWRULES.md was general contribution guidelines, so I changed the name and references in the README to reflect that fact!

Thanks!

### CODE REVIEW

**Comment by kalisjoshua** [comment]
You have reformatted the markdown to not fall within the ~80 character line limit that was being respected. Can you convert the line length limit before merging, please.

**Comment by jdpedrie** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -21,18 +21,15 @@ consistency can be more easily maintained.
...
-The basis for rules is stored in `src/defaults.json` and is the location to [add new
-rules](NEWRULES.md) as defaults. Projects using the linter to maintain
-consistency will be able to provide a customization file which will override the
-values in the defauls file.
+For detailed rules on contributions, please refer to [CONTRIBUTING.md](CONTRIBUTING.md).
 
 ## (`npm`) Scripts
 
-A number of helpful commands have been created in the npm scripts (`package.json`).
+Below is a list of commands available via NPM (`package.json`).
```
Comment (kalisjoshua) [inline README.md]: npm should be lowercase here.

**Comment by kalisjoshua** [comment]
I think if you use `git mv NEWRULES.md CONTRIBUTING.md` git should be able to recognize that it is not two seperate actions and therefore a better diff. This is looking much better than my attempt.

**Comment by jdpedrie** [comment]
Fixed the casing issue. I tried `git mv`, but it is still recognized as removing `NEWRULES.md` and adding `CONTRIBUTING.md`.

**Comment by kalisjoshua** [comment]
Good enough for me. Thank you so much.

**Comment by kalisjoshua** [comment]
One additional comment though, I would recommend you put changes you are making to a repository into a branch and not have them on master directly. It will make your life easier on more active projects potentially; if they update master and you need to rebase on those changes before your pull request is merged. Just FYI, not a problem for this repository; at this time.

**Comment by jdpedrie** [comment]
thanks for the tip!

**Pull request merged**